CHANGELOG.latest.md

```diff
@@ -1,46 +1,22 @@
-
-```
-Purchases.configure(AmazonConfiguration.Builder(this, "public_amazon_sdk_key").build())
-```
-
-For more information around configuration please take a look at the [Amazon Store section in our docs](https://docs.revenuecat.com/docs/amazon-platform-resources). The official [Amazon In-App Purchasing docs](https://developer.amazon.com/docs/in-app-purchasing/iap-overview.html) also contain very valuable information, especially around testing and best practices.
-
-If you're updating from an alpha version with Amazon Store support, remove the in-app-purchasing-2.0.76.jar dependency from your `app/build.gradle` and from the libs folder. Manually adding the jar is no longer required 😄
-
-```
-dependencies {
-    implementation files('libs/in-app-purchasing-2.0.76.jar') -> Remove this
-```
-
-**⚠️ ⚠️  Important ⚠️ ⚠️** Observer mode for Amazon is not supported yet.
- 
-### Changelog
-
-**Note:** This release is based off of 5.0.0-rc3. These are the changes compared to 5.0.0-rc3:
-
-- Removed deprecated `addAttributionData` functions and `AttributionNetwork` enum
-    https://github.com/RevenueCat/purchases-android/pull/506
-- Added retries with exponential backoff when reconnecting to Google's billing service
-    https://github.com/RevenueCat/purchases-android/pull/489
-- Updated to BillingClient 4.1
-    https://github.com/RevenueCat/purchases-android/pull/500
-- Prevent Amazon from swallowing exceptions
-    https://github.com/RevenueCat/purchases-android/pull/499
-- Improvements on Amazon implementation preventing a lot of issues related to price and currency parsing
-    https://github.com/RevenueCat/purchases-android/pull/503
-
-[Full Changelog](https://github.com/revenuecat/purchases-android/compare/main...5.0.0)
+## 5.1.0
+
+- Updated error log message for cases when there are no products configured in the dashboard 
+  - https://github.com/RevenueCat/purchases-android/pull/517
+- Added new method `setMixpanelDistinctID` as a convenience method for setting the required attribute for the Mixpanel integration
+  - https://github.com/RevenueCat/purchases-android/pull/518
+- Adds missing docs for `purchaseProduct` and `purchasePackage`
+  - https://github.com/RevenueCat/purchases-android/pull/520
+- Added `setFirebaseAppInstanceID` to allow associating RevenueCat users with Firebase.
+  - https://github.com/RevenueCat/purchases-android/pull/523
+- Adds `syncObserverModeAmazonPurchase` to sync Amazon purchases with RevenueCat. To be used in observer mode integrations.
+  - https://github.com/RevenueCat/purchases-android/pull/483
+- Better error handling when Amazon prices are missing
+  - https://github.com/RevenueCat/purchases-android/pull/527
+- Better error handling when Amazon user data cannot be loaded
+  - https://github.com/RevenueCat/purchases-android/pull/528
+- Fix detecting RevenueCat errors when loading offerings
+  - https://github.com/RevenueCat/purchases-android/pull/529
```

Comment:
Looks good! 

For iOS we've started separating API updates from other changes for the sake of readability (https://github.com/RevenueCat/purchases-ios/releases)
I'm not sure we'll be able to keep this once we also automate changelogs, though, but it's kinda nice

Reply:
I like it. I bet there's a way we can still automate it by tagging the type of change in the PR